ironfish/src/wallet/wallet.test.ts

```diff
@@ -389,6 +402,21 @@ describe('Accounts', () => {
       expect(wallet['chainProcessor']['hash']?.equals(block2.header.hash)).toBe(true)
     })
 
+    it('should not scan if wallet is not started', async () => {
```

Comment:
What's the difference between this and the test above? Is it just the `shouldRescan` check?

Reply:
yeah, that's the only difference. I'll remove the above one